charts/kubefed/charts/controllermanager/templates/deployments.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
     kubefed-control-plane: controller-manager
 {{- if .Values.controller.annotations }}
   annotations:
+    "helm.sh/hook": pre-install, pre-upgrade
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": hook-succeeded,before-hook-creation
```

Comment:
Won't `hook-succeeded` delete the hook after hook succeeds in being created?

Reply:
Reverting these changes!